pkg/resource/service.go

```diff
@@ -73,6 +73,13 @@ func (r *ServiceResource) Schema(_ context.Context, _ resource.SchemaRequest, re
 					stringplanmodifier.UseStateForUnknown(),
 				},
 			},
+			"backup_id": schema.StringAttribute{
+				Description: "ID of the backup to restore when creating new service. If specified, the service will be created as a restore operation",
+				Optional:    true,
+				PlanModifiers: []planmodifier.String{
```

Comment:
Since you block modification in ModifyPlan you don't need RequiresReplace() here.